src/cbuild/util/meson.py

```diff
@@ -33,6 +33,7 @@ def _make_crossfile(pkg, build_dir):
 objcopy = '{pkg.get_tool("OBJCOPY")}'
 pkgconfig = '{pkg.get_tool("PKG_CONFIG")}'
 llvm-config = '/usr/bin/llvm-config'
+rust = ['rustc', '--target', '{pkg.profile().triplet}', '--sysroot', '{pkg.profile().sysroot / 'usr'}']
```

Comment:
do i want to use `get_tool_flags` here? not sure

Reply:
rustflags read by rustc or cargo? i forgot

if the latter only, then probably you do..

Reply:
reading https://doc.rust-lang.org/cargo/reference/environment-variables.html#environment-variables-cargo-reads i understand that it's cargo

but also given what i said in my edit it should be fine(?)

Reply:
> edit: actually, nothing else in the binaries section has any flags, so it probably doesn't matter

ah yea; the rustc will be set, but meson will pass through whatever flags anyway i guess like it normally would. (maybe it doesn't handle rustflags at all, what happens in main/mesa?)

sure